components/table/test/table.vdiff.js

```diff
@@ -27,6 +27,48 @@ function createSortableHeaderRow() {
 		</tr>
 	`;
 }
+
+function createSortableHeaderRowWithTwoButtons() {
+	return html`
+		<tr>
+			<th>
+				<d2l-table-col-sort-button>Double 1</d2l-table-col-sort-button>
+				<d2l-table-col-sort-button>Double 2</d2l-table-col-sort-button>
+			</th>
+			<th><d2l-table-col-sort-button desc>Cell A</d2l-table-col-sort-button></th>
+			<th><d2l-table-col-sort-button nosort>Cell B</d2l-table-col-sort-button></th>
+		</tr>
+	`;
+}
+
+function createSortableHeaderRowWithTwoElements() {
+	return html`
+		<tr>
+			<th>
+				<d2l-table-col-sort-button>Double 1</d2l-table-col-sort-button>
+				<div style="display: inline-flex;">Dummy Text</div>
+			</th>
+			<th><d2l-table-col-sort-button desc>Cell A</d2l-table-col-sort-button></th>
+			<th><d2l-table-col-sort-button nosort>Cell B</d2l-table-col-sort-button></th>
+		</tr>
+	`;
+}
+
+function createSortableHeaderRowWithIconElement() {
+	return html`
+		<tr>
+			<th scope="col">
+				<d2l-table-col-sort-button>
+					Status
+				</d2l-table-col-sort-button>
+				<d2l-button-icon icon="tier1:help" type="button"></d2l-button-icon>
```

Comment:
You'll want to add an import for `button-icon` - something else is importing it, which is why this is working, but if whatever that is got removed then this test would stop working so it's good practice to have explicit imports for everything.